dex/testing/dcr/create-wallet.sh

```diff
@@ -75,12 +70,13 @@ ${SEED}
 EOF
 
 # create and unlock the wallet
-tmux new-window -t $TMUX_WIN_ID -n "w-${NAME}"
+tmux new-window -t $TMUX_WIN_ID -n "${NAME}"
 tmux send-keys -t $TMUX_WIN_ID "set +o history" C-m
 tmux send-keys -t $TMUX_WIN_ID "cd ${WALLET_DIR}" C-m
+
 echo "Creating simnet ${NAME} wallet"
-tmux send-keys -t $TMUX_WIN_ID "dcrwallet -C w-${NAME}.conf --create < wallet.answers; tmux wait-for -S w-${NAME}" C-m
-tmux wait-for w-${NAME}
+tmux send-keys -t $TMUX_WIN_ID "dcrwallet -C ${NAME}.conf --create < wallet.answers; tmux wait-for -S ${NAME}" C-m
+tmux wait-for ${NAME}
```

Comment:
The tmux bar is kinda confusing this was:

![image](https://user-images.githubusercontent.com/9373513/98572856-ba7ce080-227b-11eb-9664-131fdd238fe5.png)

Two "alphas" and two "betas"